Mathlib/Algebra/Ring/Associator.lean

```diff
@@ -0,0 +1,125 @@
+/-
+Copyright (c) 2025 Bernhard Reinke. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bernhard Reinke
+-/
+import Mathlib.Algebra.Ring.Basic
+import Mathlib.Tactic.Abel
+
+/-!
+# Associator in a ring
+
+If `R` is a non-associative ring, then  `(x * y) * z - x * (y * z)` is called the `associator` of
+ring elements `x y z : R`.
+
+The associator vanishes exactly when `R` is associative.
+
+We prove variants of this statement also for the `AddMonoidHom` bundled version of the associator,
+as well as the bundled version of `mulLeft₃` and `mulRight₃`, the multiplications `(x * y) * z` and
+`x * (y * z)`.
+-/
+
+universe u
+
+variable {R : Type u}
```

Comment:
```suggestion
variable {R : Type*}
```
is preferred, to show that one doesn't do anything weird with universes

Reply:
For just one type variable I think either is fine.